sql/core/src/main/scala/org/apache/spark/sql/execution/commands.scala

```diff
@@ -300,3 +300,72 @@ case class ShowTablesCommand(databaseName: Option[String]) extends RunnableComma
     rows
   }
 }
+
+/**
+ * A command for users to list all of the registered functions.
+ * The syntax of using this command in SQL is:
+ * {{{
+ *    SHOW FUNCTIONS
+ * }}}
+ * :: DeveloperApi ::
+ */
+@DeveloperApi
```

Comment:
Ok, I will remove it for now.
But do we have plan to expose the Expression as API for advanced developers in the future?